apps/i18n/common/en_us.json

```diff
@@ -819,11 +819,14 @@
   "levelStatus": "Level Status",
   "levelType": "Level Type",
   "libraryCodeError": "We can't publish your library because there is an error in the code. Go look for the square red error indicator and fix the errors.",
+  "libraryDescriptionPlaceholder": "Write a description of your library",
   "libraryExportNoCommentError": "This function cannot be exported until you add a comment to it.",
   "libraryExportTitle": "Export Functions as a Library",
-  "libraryName": "Library Name:",
+  "libraryName": "Library Name",
+  "libraryNameRequirements": "Your library's name must start with a capital letter and use only letters, numbers, and underscores.",
```

Comment:
note that many languages have no concept of capital letters

Reply:
I believe this came from a desire to teach good naming conventions.

Reply:
Oh you mean written/spoken languages - not programming languages.

Reply:
Great point. I'll circle back with curriculum and prod on that one.